gnosis/safe/addresses.py

```diff
@@ -522,6 +522,14 @@
         ("0xfb1bffC9d739B8D520DaF37dF666da4C687191EA", 1511, "1.3.0+L2"),
         ("0x69f4D1788e39c87893C980c06EdF4b7f686e2938", 1512, "1.3.0"),
     ],
+    EthereumNetwork.OASIS_SAPPHIRE: [
+        ("0xfb1bffC9d739B8D520DaF37dF666da4C687191EA", 330092, "1.3.0+L2"),
+        ("0x69f4D1788e39c87893C980c06EdF4b7f686e2938", 330094, "1.3.0"),
```

Comment:
@ElvisKrop thanks for your explanation here https://github.com/safe-global/safe-eth-py/pull/589#issuecomment-1641711020 My concern is not the code, because it was validated on safe-deployments PR, my concern is when was created, taking a look to your picture seems that the blocks that are harcoded here are incorrect, this block should be the creation block. I'd suggest 325640 and 325643.

Reply:
omg, sorry for that...
I was focused on the wrong topic :sweat_smile: haha
thanks for pointing! I'll update them shortly

Reply:
@moisses89 
:heavy_check_mark: blocks for oasis sapphire mainnet contracts are updated
:heavy_check_mark: I can confirm that block for testnet match the creation blocks of each contract too
:heavy_check_mark: PR branch is rebased

Hopefully, it is the final re-request for review :smile:
Thank you!